packages/ra-i18n-polyglot/src/index.ts

```diff
@@ -19,11 +19,16 @@ type GetMessages = (
  *     fr: frenchMessages,
  *     en: englishMessages,
  * };
- * const i18nProvider = polyglotI18nProvider(locale => messages[locale])
+ * const i18nProvider = polyglotI18nProvider(
+ *     locale => messages[locale],
+ *     'en',
+ *     [{ locale: 'en', name: 'English' }, { locale: 'fr', name: 'Français' }]
+ * )
  */
 export default (
     getMessages: GetMessages,
     initialLocale: string = 'en',
+    availableLocales: Locale[] = [{ locale: 'en', name: 'English' }],
     polyglotOptions: any = {}
 ): I18nProvider => {
     let locale = initialLocale;
```

Comment:
Shouldn't we throw a warning if the `initialLocal` is not in the `availableLocales`? Maybe I'm nitpicking here